### DISCUSSION TITLE
Updated IsaacLab Benchmark results

### DISCUSSION THREAD

**Discussion by fl1ps1de5** [Toni-SM/skrl#209] [Ideas]
Hi there,

Just wondering if there is some updated benchmarking results for the IsaacLab environments.

Clearly, rewards are scaled a bit differently to previous Isaac workflows (i.e. Cartpole in IsaacLab is solved with ~5 reward, compared to ~500 reward in Isaac Orbit).

On the skrl docs, the provided link only goes to Isaac Orbit results.

Just wondering if this information is available (specifically wondering about the Ant and Reach tasks...)

Regards,
Fl1ps1de5

**Comment by Toni-SM** [comment]
Hi @fl1ps1de5 

Not yet, but there is a work in progress in the `toni/isaaclab_examples` branch to update the Isaac Lab examples, benchmark and checkpoints
